patterns/footer-portfolio.php

```diff
@@ -7,13 +7,14 @@
  */
 ?>
 
-<!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"7vh","bottom":"7vh"}}}} -->
-<div class="wp-block-group alignwide" style="padding-top:7vh;padding-bottom:7vh"><!-- wp:image {"width":"40px","height":"auto","sizeSlug":"full","linkDestination":"none"} -->
-<figure class="wp-block-image size-full is-resized"><img src="<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/icon-message.webp" alt="" style="width:40px;height:auto"/></figure>
```

Comment:
the invalid warning with the image was a change that is not on 6.3 but will be on 6.4

Reply:
basically aspect ratio

Reply:
Ah of course - I've reverted to the previous version for that figure.